kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -70,8 +70,168 @@
     </KGrid>
 
     <hr class="bottom-border">
+    <div v-if="isQuestionAvailable">
+      <KGrid
+        class="question-row"
+      >
+        <KGridItem
+          :layout12="{ span: 6 }"
+        >
+          <div class="left-column-alignment-style">
+            <div>
+              <p>
+                <KCheckbox />
+              </p>
+            </div>
+
+            <div>
+              <p>{{ $tr('selectAllLabel') }}</p>
+            </div>
+          </div>
+        </KGridItem>
+
+        <KGridItem
+          :layout12="{ span: 6 }"
+        >
+          <div class="right-alignment-style">
+            <KGrid>
+              <KGridItem :layout12="{ span: 4 }">
+                <div class="icon-container">
+                  <KIcon class="reduce-chervon-spacing" icon="chevronDown" />
+                  <KIcon class="reduce-chervon-spacing" icon="chevronUp" />
+                </div>
+              </KGridItem>
+
+              <KGridItem :layout12="{ span: 4 }">
+                <KIcon
+                  class="icon-size"
+                  icon="refresh"
+                />
+              </KGridItem>
+
+              <KGridItem :layout12="{ span: 4 }">
+                <KIcon
+                  class="icon-size"
+                  icon="trash"
+                />
+              </KGridItem>
+            </KGrid>
+          </div>
+        </KGridItem>
+
+      </KGrid>
+
+      <AccordionContainer
+        :items="placeholderList"
+      >
+        <template
+          #default="{ isItemExpanded, toggleItemState }"
+        >
+          <div
+            v-for="item in placeholderList"
+            :key="item.id"
+          >
```

Comment:
One thing that came to mind while testing was whether or not keyboard nav focus should cycle through each piece to get to the `>` KIconButton to open it so I went and tested the [W3](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/) example and noticed that the focus flow goes like:

https://github.com/learningequality/kolibri/assets/6356129/81010bc6-714f-4d39-a576-0b626d87ad32

However, ours instead of highlighting the whole "header" bit, goes into the children of the row (drag handle -> checkbox -> open/close icon).

Our situation _is_ a bit different because we have interactions within the heading (drag/checkbox) whereas the example above only has text. 

This is a little bit of a pickle... because we also want clicking anywhere on the header to toggle open/close and not just the iconbutton - in fact, I think that that KIconButton should really just be a KIcon. **BUT** the issue might be in that when you click to select a KCheckbox, you'll also end up clicking to trigger the toggle open/close. 

For follow up here I think that maybe the next step is to test out what a `@click` handler here might work okay without getting triggered while clicking the drag handle or KCheckbox. We may need to make some decisions based on how that goes. I'll be happy to cohack a bit on this with you as well to see what we can work out.